lib/mongoose_push_web/controllers/api_v1_controller.ex

```diff
@@ -0,0 +1,34 @@
+defmodule MongoosePushWeb.APIv1Controller do
+  alias MongoosePushWeb.Schemas
+  alias OpenApiSpex.Operation
+  use MongoosePushWeb, :controller
+
+  @spec open_api_operation(atom) :: Operation.t()
+  def open_api_operation(action) do
+    operation = String.to_existing_atom("#{action}_operation")
+    apply(__MODULE__, operation, [])
+  end
+
+  @spec handle_operation() :: Operation.t()
+  def handle_operation() do
+    %Operation{
+      tags: ["apiv1"],
+      summary: "sends a push",
+      description: "performs the sending of push notification",
+      operationId: "APIv1Controller.handle",
+      parameters: [
```

Comment:
I'm pretty sure we don't need that - more research needed.

Reply:
I'm not sure parameters in the request's path can be described in a different way. At least, I didn't find it defined differently in the lib's doc.

Reply:
From my understanding, the schemas can be used to define the request body. Other parameters like these in `path`, `query`, `header` or `cookie` are described with thise API: https://hexdocs.pm/open_api_spex/OpenApiSpex.Operation.html#parameter/5

Reply:
Sorry, I just saw it :) it is possible to pass the schema reference to the `Operation.parameter` function as 3rd parameter. The question is do we want that? On one hand, this probably common for all versions of the IP, on the other hand the type is just a string.

Reply:
No, no, why do we even need parameters? Like you said - those are to define `path`, `query`, `header` or `cookie`, none of which MongoosePush uses.

Reply:
`parameters` is just a HTTP's "form data" encoded requests. In case of GET those are implied, but we deal with POST only in which case, this is just one of possible encodings. Not only that, it's also handled by more generic `request_body`: https://swagger.io/docs/specification/describing-request-body/

Reply:
MongoosePush uses JSON "officially" and only that. HTTP's form data could be _allowed_ for APIv1, as it's flat anyway, but for APIv3 that's not even an option.

Reply:
Ok, we have the path 'v1/notification/:device_id'
Do you mean that even though the device_id is part of path, it can be described in request_body ?

Reply:
Oh wow, so sorry about that - the file preview "collapsed" a bit and I misread the implementation. No, you're completely right - we do have one query parameter and it should be handled the way it is. Again, sorry :)